src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeControllerTest.java

```diff
@@ -582,16 +583,36 @@ public void testGetDisputeUpdateRequests_200() throws Exception {
 		List<DisputeUpdateRequest> results = new ArrayList<DisputeUpdateRequest>();
 		results.add(disputeUpdateRequest2);
 
-		// Mock underlying findDisputeUpdateRequestByDisputeIdAndStatus service
-		Mockito.when(service.findDisputeUpdateRequestByDisputeIdAndStatus(dispute1.getDisputeId(), DisputeUpdateRequestStatus.ACCEPTED)).thenReturn(results);
-
 		// issue a GET request, expect 200 and correct ID and status
+		Mockito.when(service.findDisputeUpdateRequestByDisputeIdAndStatus(dispute1.getDisputeId(), DisputeUpdateRequestStatus.ACCEPTED)).thenReturn(results);
 		ResponseEntity<List<DisputeUpdateRequest>> controllerResponse = disputeController.getDisputeUpdateRequests(dispute1.getDisputeId(), DisputeUpdateRequestStatus.ACCEPTED);
 		List<DisputeUpdateRequest> resultList = controllerResponse.getBody();
 		assertNotNull(resultList);
 		assertEquals(disputeUpdateRequest2.getDisputeUpdateRequestId(), results.get(0).getDisputeUpdateRequestId());
 		assertEquals(disputeUpdateRequest2.getStatus(), results.get(0).getStatus());
 		assertEquals(controllerResponse.getStatusCode().value(), HttpStatus.OK.value());
+		
+		// Get all update requests with null params
+		Mockito.when(service.findDisputeUpdateRequestByDisputeIdAndStatus(null, null)).thenReturn(results);
+		controllerResponse = disputeController.getDisputeUpdateRequests(null, null);
+	    resultList = controllerResponse.getBody();
+		assertNotNull(resultList);
+		assertEquals(resultList.size(),4);
+				
+    	// Get all update requests with pending status
+		Mockito.when(service.findDisputeUpdateRequestByDisputeIdAndStatus(null, DisputeUpdateRequestStatus.PENDING)).thenReturn(results);
+		controllerResponse = disputeController.getDisputeUpdateRequests(null, DisputeUpdateRequestStatus.PENDING);
+		resultList = controllerResponse.getBody();
+		assertNotNull(resultList);
+		assertEquals(resultList.size(),2);
+				
+    	// Get all update requests for given dispute id
+		Mockito.when(service.findDisputeUpdateRequestByDisputeIdAndStatus(dispute1.getDisputeId(), null)).thenReturn(results);
+		controllerResponse = disputeController.getDisputeUpdateRequests(dispute1.getDisputeId(), null);
+	    resultList = controllerResponse.getBody();
+	    assertNotNull(resultList);
+		assertEquals(resultList.size(),2);
+		assertEquals(resultList.get(0).getDisputeId(), dispute1.getDisputeId());						

```

Comment:
These tests may not be working because what's returned by `service.findDisputeUpdateRequestByDisputeIdAndStatus()` method is just mocked by adding some specific data to the `List<DisputeUpdateRequest> results` list. It's not actually calling any database. So, you need to modify what's added to the `results` list for asserting some values based on what you're passing into the controller and expected result.